envbuilder.go

```diff
@@ -586,117 +652,130 @@ func Run(ctx context.Context, opts options.Options) error {
 			}
 			opts.Logger(log.LevelInfo, "#%d: 👀 Found devcontainer.json label metadata in image...", stageNumber)
 			for _, container := range devContainer {
-				if container.RemoteUser != "" {
-					opts.Logger(log.LevelInfo, "#%d: 🧑 Updating the user to %q!", stageNumber, container.RemoteUser)
+				if container.ContainerUser != "" {
+					opts.Logger(log.LevelInfo, "#%d: 🧑 Updating the user to %q!", stageNumber, container.ContainerUser)
 
-					configFile.Config.User = container.RemoteUser
+					configFile.Config.User = container.ContainerUser
```

Comment:
Review: Decided to at least correct the behavior here. The remoteUser must not affect the user we run the container as, it's for devcontainer services & co.